x-pack/legacy/plugins/actions/server/actions_config.ts

```diff
@@ -65,14 +82,26 @@ function isWhitelistedHostnameInUri(config: ActionsConfigType, uri: string): boo
   );
 }
 
+function isActionTypeEnabledinConfig(
+  { enabledTypes }: ActionsConfigType,
+  actionType: string
+): boolean {
+  const enabled = new Set(enabledTypes);
+  if (enabled.has('*')) return true;
+  if (enabled.has(actionType)) return true;
```

Comment:
I like how this is implemented, so, as you're already in here, it might be worth refactoring `isWhitelisted` to use the same `Set` based method (we use `find` there to avoid cycling through all the values every time, which is what happens when you create a `Set` but the code is so much simpler and easy to maintain that it seems worth it).

:+1:

Reply:
I'll see, I was thinking about that, but for some reason thought it wouldn't be applicable to the whitelist.

Reply:
`Set` is one of the best things added to JS, evar.  Changing wl to use the same pattern, thx!